Plugins/ServerLogRedirector/ServerLogRedirector.cpp

```diff
@@ -34,6 +32,8 @@ using namespace NWNXLib;
 using namespace NWNXLib::API;
 using namespace NWNXLib::Services;
 
+static bool m_WriteTimestampedLogEntry;
```

Comment:
Nitiest of nits: `m_` is used for member fields. Statics sometimes use `s_`, or just no prefix at all.